server/lobbyconnection.py

```diff
@@ -715,8 +747,17 @@ def _set_user_agent_and_version(self, user_agent, version):
 
             async with self._db.acquire() as conn:
                 result = await conn.execute(
-                    "SELECT url, tooltip FROM `avatars` "
-                    "LEFT JOIN `avatars_list` ON `idAvatar` = `avatars_list`.`id` WHERE `idUser` = %s", (self.player.id,))
+                    select([
+                        avatars_list.c.url,
+                        avatars_list.c.tooltip
+                    ]).select_from(
+                        avatars.outerjoin(
+                            avatars_list
```

Comment:
You better give conditions

Reply:
No, SQLAlchemy does this automatically. That's the whole point.